src/modules/requirements/QuestLineStartedRequirement.ts

```diff
@@ -0,0 +1,23 @@
+import { AchievementOption } from '../GameConstants';
+import QuestLineState from '../quests/QuestLineState';
+
+import Requirement from './Requirement';
+
+export default class QuestLineStartedRequirement extends Requirement {
+    questLineName: string;
+
+    constructor(questLineName: string, option = AchievementOption.equal) {
+        super(1, option);
+        this.questLineName = questLineName;
+    }
+
+    public getProgress(): number {
+        return (App.game.quests.getQuestLine(this.questLineName).state() === QuestLineState.started
```

Comment:
Are we able to bind `App.game.quests.getQuestLine(this.questLineName)` to something in the constructor so we don't have to keep on calling it here?

Reply:
I could not get it to work. If i remember correctly, the quest lines are not created there. But might be able to add some caching. Might want to do it in a new PR, as i can do the same for other requirements